src/P4est.jl

```diff
@@ -5,4 +5,6 @@ using Reexport: @reexport
 
 include("libp4est.jl")
 
+uses_mpi() = isdefined(@__MODULE__, :P4EST_ENABLE_MPI)
```

Comment:
Ah, wait a minute. I forgot to ask: Do you want this function to be exported or should it be an internal function? Right now, it is not exported. If it is exported, it should get a docstring.

Reply:
I think it shouldn't be exported but should be used as `P4est.uses_mpi()`. But I would consider it part of the public API since it's going to be used in Trixi which is a different package so I guess there should be a docstring and mention in the README. Do you agree?

Reply:
Done in 40d2220.

Reply:
Thanks!